release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -390,6 +390,13 @@ Be aware that CIDR ranges for networks are not adjustable after cluster installa
 
 If you are using the OVN-Kubernetes cluster network provider, you can now enable IPsec encryption after cluster installation. For more information about how to enable IPsec, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
 
+[id="ocp-4-11-support-for-hosted-control-planes"]
+==== Hosted control planes (Technology Preview)
+
+With hosted control planes for {product-title}, you can host clusters at scale to reduce management costs, optimize cluster deployment time, and separate management and workload concerns. You can enable this deployment model as a Technology Preview feature when you install the multicluster engine for Kubernetes operator version 2.0. For more information, see xref:../architecture/control-plane.html#hosted-control-planes-overview_control-plane[Overview of hosted control planes (Technology Preview)].
```

Comment:
Just wondering if this is the actual proper name of the Operator: `multicluster engine for Kubernetes operator`? 

Usually I would think that there should be some capitalization in there. Can you check on that (if you haven't already)?

Reply:
I'm pretty sure I've seen it in lowercase in the MCE and ACM docs, but I will double check.

Reply:
I verified that the correct capitalization is as follows: "multicluster engine for Kubernetes".